torchvision/prototype/features/_feature.py

```diff
@@ -92,6 +92,7 @@ def __torch_function__(
             return NotImplemented
 
         with DisableTorchFunction():
+            raise Exception(str(func), str(args), str(kwargs))
```

Comment:
This looks like something added for debugging?
```suggestion
```

Reply:
Ah yes, sorry too many things at once :facepalm: